etc/workflows/duplicate-event.xml

```diff
@@ -12,18 +12,48 @@
     <![CDATA[
       <div id="workflow-configuration">
         <fieldset>
-          <input
-            id="numberOfEvents"
-            name="numberOfEvents"
-            type="number"
-            class="configField"
-            onkeypress="return event.charCode > 47"
-            oninput="checkValueInBounds()"
-            min="1"
-            value="1"
-            max="25"
+          <br/>
+          <div>
+            <label for="mpTitle" style="width: 150px;" >Title of Duplicate:</label>
+            <input id="mpTitle" name="mpTitle" type="text" style="width: 400px;" value=" " class="configField" />
+          </div>
+          <br/>
+          <div>
+            <label for="seriesId" style="width: 150px;" >Series Id:</label>
+            <input id="seriesId" name="seriesId" type="text" maxlength="255" style="width: 400px;" value=" " class="configField" />
+          </div>
+          <br/>
+          <div>
+            <label for="startDate" style="width: 150px;">StartDate (YYYY-MM-DD):</label>
+            <input id="startDate" name="startDate" type="text" style="width: 200px;" value=" " class="configField" />
```

Comment:
Use [`<input type="date">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date)

Reply:
Yeah, I used date as a type before but the trouble is you can't really set a default value as `value =""` (empty). 
So when the date is not set, a null value is sent for that property which in turn throws an exception.
Properties values are enforced in `org.opencastproject.assetmanager.impl.persistence` package by the following code:

```
  public Property toProperty() {
    final PropertyId id = new PropertyId(mediaPackageId, namespace, propertyName);
    if (stringValue != null) {
      return new Property(id, Value.mk(stringValue));
    } else if (dateValue != null) {
      return new Property(id, Value.mk(dateValue));
    } else if (longValue != null) {
      return new Property(id, Value.mk(longValue));
    } else if (boolValue != null) {
      return new Property(id, Value.mk(boolValue));
    } else {
      throw new RuntimeException("Bug. At least one of the value columns must be non null.");
    }
  }
```
Basically input of type date or time can't be used unless a default value is set. Or maybe you have some suggestions.

Reply:
In that case, where the time is null, I suggest using the current time and log a warning for the event.

Reply:
fixed, current datetime set default in UI.